### PR TITLE
[doc] PR for Include YSQL Port 13000 for Firewall in Platform Installation

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-3352

Preview: 
https://deploy-preview-11747--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws/
https://deploy-preview-11747--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 1]
```diff
@@ -60,28 +60,33 @@ showAsideToc: true
...
-- View the Yugabyte Platform console (port `tcp:80`)
+- View the Yugabyte Platform UI (port `tcp:80`)
 
-If you are using your own custom VPCs (self-managed configuration), the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379. For more information on ports used by YugabyteDB, refer to [Default ports](../../../../reference/configuration/default-ports).
+If you are using your own Virtual Private Cloud (VPC) as a self-managed configuration, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 13000, 9300, 9042, 5433, 6379. For more information on ports used by YugabyteDB, refer to [Default ports](../../../../reference/configuration/default-ports).
 
-To create a security group that enables these, go to **EC2 > Security Groups**, click **Create Security Group** and then add the following values:
+To create a security group that enables these, use the Amazon console to navigate to **EC2 > Security Groups**, click **Create Security Group**, and then add the following values:
+
+- Enter a meaningful name, such as, for example, `yugaware-sg`.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:73]:
```suggestion
- Enter a meaningful name (for example, yugaware-sg).
```

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 2]
```diff
@@ -60,28 +60,33 @@ showAsideToc: true
...
 
-- For the name, enter `yugaware-sg` (you can change the name if you want).
 - Add a description (for example, `Security group for Yugabyte Platform access`).
-- Add the appropriate IP addresses to the **Source IP ranges** field. To allow access from any machine, add `0.0.0.0/0` but note that this is not very secure.
-- Add the ports `22`, `8800`, and `80` to the **Port Range** field. The **Protocol** selected must be `TCP`. For a self-managed configuration, also add the 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379 TCP ports.
 
-You should see something like the screenshot below. Click **Create** next.
+- Add the appropriate IP addresses to the **Source IP ranges** field. To allow access from any computer, add `0.0.0.0/0` but note that this is not very secure.
+
+- Add the ports `22`, `8800`, and `80` to the **Port Range** field. The **Protocol** selected must be `TCP`.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:79]:
```suggestion
- Add the ports 22, 8800, and 80 to the **Port Range** field. The **Protocol** selected must be TCP.
```

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 3]
```diff
@@ -60,28 +60,33 @@ showAsideToc: true
...
+If you are using your own Virtual Private Cloud (VPC) as a self-managed configuration, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 13000, 9300, 9042, 5433, 6379. For more information on ports used by YugabyteDB, refer to [Default ports](../../../../reference/configuration/default-ports).
 
-To create a security group that enables these, go to **EC2 > Security Groups**, click **Create Security Group** and then add the following values:
+To create a security group that enables these, use the Amazon console to navigate to **EC2 > Security Groups**, click **Create Security Group**, and then add the following values:
+
+- Enter a meaningful name, such as, for example, `yugaware-sg`.
 
-- For the name, enter `yugaware-sg` (you can change the name if you want).
 - Add a description (for example, `Security group for Yugabyte Platform access`).
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:75]:
```suggestion
- Add a description (for example, Security group for Yugabyte Platform access).
```

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 4]
```diff
@@ -60,28 +60,33 @@ showAsideToc: true
...
-- Add the appropriate IP addresses to the **Source IP ranges** field. To allow access from any machine, add `0.0.0.0/0` but note that this is not very secure.
-- Add the ports `22`, `8800`, and `80` to the **Port Range** field. The **Protocol** selected must be `TCP`. For a self-managed configuration, also add the 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, and 6379 TCP ports.
 
-You should see something like the screenshot below. Click **Create** next.
+- Add the appropriate IP addresses to the **Source IP ranges** field. To allow access from any computer, add `0.0.0.0/0` but note that this is not very secure.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:77]: Don't see Source IP ranges field in screenshot - supposed to click Add Rule at some point?

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 5]
```diff
@@ -140,7 +145,7 @@ In order for Yugabyte Platform to manage YugabyteDB nodes, limited access to you
 
 ## Provision an instance for Yugabyte Platform
 
-Create an instance to run the Yugabyte Platform server. To do this, navigate to **EC2 > Instances**, click **Launch Instance**, and enter the following values:
+You need to create an instance to run the Yugabyte Platform server. To do this, navigate to **EC2 > Instances**, click **Launch Instance**, and enter the following values:
 
 - Change the boot disk image to Ubuntu Server 16.04, as shown in the following illustration: <br><br>
 ![Image](/images/ee/aws-setup/yugaware-create-instance-os.png)
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:151]: Link Create an IAM role below is broken

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 6]
```diff
@@ -60,28 +60,33 @@ showAsideToc: true
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:60]: Maybe set isTocNested: to true to show additional headings in TOC

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md [hunk 7]
```diff
@@ -189,15 +194,15 @@ If you are configuring an existing instance of Yugabyte Platform, start by attac
...
+Finally, execute the following command to change metadata options (replace `NNNNNNN` with the instance ID and `us-west-2` with the region in which this EC2 VM is deployed):
 
 ```shell
 aws ec2 modify-instance-metadata-options --instance-id i-NNNNNNN --http-put-response-hop-limit 3 --http-endpoint enabled --region us-west-2
 ```
 
 For more information, see [Configure the instance metadata service](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html).
 
-When you create an AWS cloud provider, as described in [Configure the AWS cloud provider](../../../configure-yugabyte-platform/set-up-cloud-provider/aws/), you need to  complete the following fields in the **Cloud Provider Configuration > AWS** screen:
+When you create an AWS cloud provider, as described in [Configure the AWS cloud provider](../../../configure-yugabyte-platform/set-up-cloud-provider/aws/), you need to  complete the following fields in the **Configs > Cloud Provider Configuration > AWS** page of of Yugabyte Platfrom UI :
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:205]:
```suggestion
When you create an AWS cloud provider, as described in [Configure the AWS cloud provider](../../../configure-yugabyte-platform/set-up-cloud-provider/aws/), you need to  complete the following fields in the **Configs > Cloud Provider Configuration > AWS** page of of Yugabyte Platform UI :
```

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md [hunk 8]
```diff
@@ -96,13 +96,13 @@ For more information, see [Creating and managing service accounts](https://cloud
 
 ## Create a firewall rule
 
-In order to access Yugabyte Platform from outside the GCP environment, you have to enable firewall rules. At a minimum, you need the following:
+In order to access Yugabyte Platform from outside the GCP environment, you have to enable firewall rules. At a minimum, you need to be able to do the following:
 
-- Access the Yugabyte Platform instance over SSH (port `tcp:22`)
-- Check, manage, and upgrade Yugabyte Platform (port `tcp:8800`)
-- View the Yugabyte Platform UI (port `tcp:80`)
+- Access the Yugabyte Platform instance over SSH (port `tcp:22`).
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md:101]: On AWS page you didn't add periods to this list

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md [hunk 9]
```diff
@@ -96,13 +96,13 @@ For more information, see [Creating and managing service accounts](https://cloud
...
+- Check, manage, and upgrade Yugabyte Platform (port `tcp:8800`).
+- View the Yugabyte Platform UI (port `tcp:80`).
 
-If you are using your own Virtual Private Cloud (VPC) as a self-managed configuration, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, 5433, 6379. For more information, see [Default ports](../../../../reference/configuration/default-ports).
+If you are using your own Virtual Private Cloud (VPC) as a self-managed configuration, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 13000, 9300, 9042, 5433, 6379. For more information, see [Default ports](../../../../reference/configuration/default-ports).
 
 Next, you need to create a firewall entry, as follows:
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md:108]: In Provision section, the stuff you enter is in code blocks, seems overused

Final sentence says yugaware-1-gcp, but the example is yugabyte-1-gcp

**Review by ddhodge — APPROVED** [review]
Misc comments

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:73]: changed

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:79]: changed

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:77]: This field is called "Source". Changed.

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md (hunk 7 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md:205]: fixed

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md (hunk 8 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md:101]: Added periods

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md (hunk 9 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md:108]: changed

**Pull request merged**